src/dbnode/persist/fs/merger.go

```diff
@@ -128,6 +130,8 @@ func (m *merger) Merge(
 		NamespaceMetadata: nsMd,
 		Shard:             shard,
 		BlockStart:        startTime,
+		VolumeIndex:       fileID.VolumeIndex + 1,
```

Comment:
I think it would be a little cleaner to have this passed in (instead of just incing) so that its very explicit what volume number you're trying to create.

Reply:
Sure.